iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/import_/Importers.java

```diff
@@ -280,18 +280,36 @@ public static Importer removePostProcessors(Importer importer) {
      * @param dataSource data source
      * @param parameters some properties to configure the import
      * @param computationManager computation manager to use for default post processors
+     * @param config the import configuration
+     * @param reporter the reporter used for functional logs
      * @return the model
      */
-    public static Network importData(ImportersLoader loader, String format, ReadOnlyDataSource dataSource, Properties parameters, ComputationManager computationManager, ImportConfig config) {
```

Comment:
You should still keep this signature method calling Reporter.NO_OP

Reply:
Yes, the signature with all previous parameters is being kept, and calls this one with Reporter.NO_OP. It is the variant just below this one in the new code. I choose to update this lines to include the parameter instead of adding new lines in another part of the code because this is the only variant that is completely documented